ajax/libs/graingert-wow/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "graingert-wow",
+  "filename": "wow.min.js",
+  "title": "WOW",
```

Comment:
Sorry for late response. @x09326  Because that's OK, we can rebase first and then call Peter.